### PR TITLE
PATH updated in install scripts for Centos 6.7

### PR DESCRIPTION
**Pull request by mwollenweber**
Some versions of Centos 6.7 have installation problems related to $PATH not including necessary directories. To fix this issue, I defined $PATH at the start of the RHEL section the default in my VM.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -13,6 +13,7 @@ SCRIPTS="$MHN_HOME/scripts/"
 cd $SCRIPTS
 
 if [ -f /etc/redhat-release ]; then
+    export PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
```
Comment (jt6211) [inline install.sh]: IMO, you should not do this.  It should be something like:

```
export PATH=/usr/local/sbin:/usr/local/bin:$PATH
```

Resetting the whole PATH may have unintended consequences.

Comment (mwollenweber) [inline install.sh]: I debated this. PATH inside the script shouldn't affect anything outside the script. Setting it to the clean default setting seemed best. In most environments appending the existing $PATH will just duplicate the path making it $PATH=$PATH:$PATH. But both work. I'll make the change.

Comment (jt6211) [inline install.sh]: if you test this and it works on redhat and ubuntu that is fine.

**Comment by mwollenweber** [comment]
The non-default install scripts haven't been updated to include code for RHEL. If that's needed, I should be able to quickly do that.

**Comment by jt6211** [comment]
yeah i think we want the non default scripts to have this too so people don't run into the same issue we saw yesterday where supervisor fails due to /usr/local/bin not being on the path.

**Pull request merged**